server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/DelegateSetContract.scala

```diff
@@ -574,4 +580,266 @@ trait DelegateSetContract {
            |	}
            |}""".stripMargin)
   }
+
+  @Test
+  def destroyShouldSucceed(server: GuiceJamesServer) : Unit = {
+    server.getProbe(classOf[DelegationProbe])
+      .addAuthorizedUser(BOB, ANDRE)
+
+    val delegationId = DelegationId.from(BOB, ANDRE).serialize
+
+    val request: String =
+      s"""{
+         |	  "using": ["urn:ietf:params:jmap:core", "urn:apache:james:params:jmap:delegation"],
+         |    "methodCalls": [
+         |      [
```

Comment:
indent issue? and other places

Reply:
no, it still is pretty
<img width="967" alt="image" src="https://user-images.githubusercontent.com/81145350/213078437-85183073-729f-4f86-899c-d04860ff728e.png">